src/operator-docs/transformation/windowToggle.ts

```diff
@@ -1,6 +1,102 @@
 import { OperatorDoc } from '../operator.model';
 
 export const windowToggle: OperatorDoc = {
-  'name': 'windowToggle',
-  'operatorType': 'transformation'
+  name: 'windowToggle',
+  operatorType: 'transformation',
+  signature:
+    'public windowToggle(openings: Observable, closingSelector: function(value: O): Observable): Observable',
+  marbleUrl: 'http://reactivex.io/rxjs/img/windowToggle.png',
+  parameters: [
+    {
+      name: 'openings',
+      type: 'Observable<O>',
```

Comment:
maybe remove this generic. Could be confusing

Reply:
hmm, will remove it.